chart/values.yaml

```diff
@@ -705,6 +705,13 @@ obs:
         # NodePort associated with https port
         https: 90010
 
+storageClass:
+  enabled: true
+  nameSuffix: single-replica
+  parameters:
+    default: false
```

Comment:
@tiagolobocastro @synthe102 Should this also not be a part of the parameters?

Reply:
oh I missed that, yes that should also not be on parameters

Reply:
Could you also move this @synthe102 ? Thanks